edx_exams/apps/core/management/commands/bulk_add_course_staff.py

```diff
@@ -0,0 +1,72 @@
+"""Management command to populate the CourseStaffRole model and related User objects, from LMS"""
+import logging
+
+import unicodecsv
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from edx_exams.apps.core.models import (
+    CourseStaffRole,
+    User,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    todo: add docs here
+    """
+    help = 'todo: Add help here'
+
+    def add_arguments(self, parser):
+        # todo: should csv path be required?
+        parser.add_argument(
+            '-p', '--csv_path',
+            metavar='csv_path',
+            dest='csv_path',
+            required=True,
+            help='Path to CSV file.')
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        """
+        todo: Main logic
+        """
+        csv_path = options['csv_path']
+
+        with csv_path:
+            with open(csv_path, 'rb') as csv_file:
+                self.add_course_staff_from_csv(csv_file)
+
+        logger.info('Bulk add course staff complete!')
+
+    def add_course_staff_from_csv(self, csv_file):
+        """
+        Add the given set of course staff provided in csv
+        """
+        reader = list(unicodecsv.DictReader(csv_file))
+        for row in reader:
+            self.add_course_staff(row.get('username'), row.get('email'), row.get('role'))
```

Comment:
you'll want to build up blocks of multiple rows and do a bulk add to avoid creating a ton of database transactions.